libs/libtalloc/libtalloc-cache.txt

```diff
@@ -0,0 +1,39 @@
+Checking simple C program: OK
```

Comment:
libtalloc uses wafsamba to configure and build.
It cannot cross-compiling whithout either --cross-execute (emulator) or --cross-answers (which contains the configuration answers for the target system). https://wiki.samba.org/index.php/Waf#cross-compiling
The file libtalloc-cache.txt contains theses answers. I do not see other options to build libtalloc.